scout/server/blueprints/variant/controllers.py

```diff
@@ -462,7 +462,8 @@ def get_loqusdb_obs_cases(
     obs_cases = []
     user_institutes_ids = set([inst["_id"] for inst in user_institutes(store, current_user)])
     for i, case_id in enumerate(obs_families):
-        if len(obs_cases) == 10:
+        max_observations = 50
+        if len(obs_cases) == max_observations:
```

Comment:
```suggestion
        if len(obs_cases) == 50:
```

Without adding a new variable seems easier, no?

Reply:
Matter of style I guess, I tend to be a bit more verbose and use named variables to make it clear what the purpose of each variable is.

But let's stick with your preferred approach here. I'll update!